### PR TITLE
reactivate ec website test as last website being tested

### PR DESCRIPTION
**Pull request by moving-bits**
Do not merge for now, due to https://github.com/cgeo/cgeo/issues/7732#issuecomment-632822577

- reactivate ec website test
- but move it to the end of the website tests to not block other website tests

### CODE REVIEW

**Comment by Lineflyer** [comment]
Using a non htttps page here does not really make sense.
In fact I would like the test to fail if their certificate is not ok, as this means, that also our users cant use the API.

File: main/src/cgeo/geocaching/connector/ec/ECConnector.java [hunk 1]
```diff
@@ -86,6 +86,15 @@ public String getNameAbbreviated() {
         return "EC";
     }
 
+    @Override
+    @NonNull
+    public String getTestUrl() {
```
Comment (bekuno) [inline main/src/cgeo/geocaching/connector/ec/ECConnector.java:91]: The test data should be in the test code.

**Comment by moving-bits** [comment]
> Using a non htttps page here does not really make sense.
> In fact I would like the test to fail if their certificate is not ok, as this means, that also our users cant use the API.

Ok, then I'd misunderstood you. I removed the last commit (non https-URL), so that the PR now only consists of reactivating the EC website test, but as the last website being tested.

File: main/src/cgeo/geocaching/connector/ec/ECConnector.java (hunk 1 above)
Comment (moving-bits) [reply main/src/cgeo/geocaching/connector/ec/ECConnector.java]: That's the way the `IConnector` interface is implemented currently (and used by the `GCConnector` as well).

But anyway, it's obsolete now, as I I've removed the addition of an individual testUrl for the EC connector due to the change requested by @Lineflyer.

**Pull request merged**